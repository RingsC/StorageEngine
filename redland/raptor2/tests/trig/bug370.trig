# Issue #0000370 http://bugs.librdf.org/mantis/bug_change_status_page.php
#
# rapper -i trig --show-graphs bug370.trig http://mmt.me.uk/examples/rdf/trig1.tt
#

@base <http://mmt.me.uk/examples/rdf/trig1.ttl> .

@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
@prefix foaf: <http://xmlns.com/foaf/0.1/> .
@prefix foo: <http://example.com/foo/> .
@prefix : <http://example.com/lame/> .

:romeo a foaf:Person ;
    foaf:name "Romeo Montague".
    
:juliet a foaf:Person ; 
    foaf:name "Juliet Capulet" .

:juliet foaf:gender "female" .

:reality { 
    :romeo foo:loves :juliet . } 

:alternative { 
    :romeo foo:hates :juliet . }

:romeo foaf:gender "male" .   
