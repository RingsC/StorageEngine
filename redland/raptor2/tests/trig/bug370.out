<http://example.com/lame/romeo> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://xmlns.com/foaf/0.1/Person> .
<http://example.com/lame/romeo> <http://xmlns.com/foaf/0.1/name> "Romeo Montague" .
<http://example.com/lame/juliet> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://xmlns.com/foaf/0.1/Person> .
<http://example.com/lame/juliet> <http://xmlns.com/foaf/0.1/name> "Juliet Capulet" .
<http://example.com/lame/juliet> <http://xmlns.com/foaf/0.1/gender> "female" .
<http://example.com/lame/romeo> <http://example.com/foo/loves> <http://example.com/lame/juliet> <http://example.com/lame/reality> .
<http://example.com/lame/romeo> <http://example.com/foo/hates> <http://example.com/lame/juliet> <http://example.com/lame/alternative> .
<http://example.com/lame/romeo> <http://xmlns.com/foaf/0.1/gender> "male" .
